beacon_node/beacon_chain/src/block_verification.rs

```diff
@@ -282,7 +287,11 @@ pub enum BlockError<T: EthSpec> {
     /// If it's actually our fault (e.g. our execution node database is corrupt) we have bigger
     /// problems to worry about than losing peers, and we're doing the network a favour by
     /// disconnecting.
-    ParentExecutionPayloadInvalid { parent_root: Hash256 },
+    ParentExecutionPayloadInvalid {
+        parent_root: Hash256,
+    },
+    PublishError,
```

Comment:
This is a remnant of my branch (sorry 😊) , but having `PublishError` a variant of `BlockError` is annoying because otherwise `BlockError` just covers block validation and directly maps to different peer scoring scenarios.  Could we make it to a `BeaconChainError` variant instead?  This shouldn't change too much beacuse the `BeaconChainError` is already nested in the `BlockError`

Reply:
Resolved in 0066e6e8631c3d1c290cd817e998242c27ae726d